docs/transfer/3-3-1-bad-quote-request.puml

```diff
@@ -0,0 +1,127 @@
+@startuml
+
+title Transfer: 3.3.1 Bad Quote Request
+
+
+box "PISP"
+participant "PISP Server" as D1
+end box
+box "Mojaloop"
+    participant Switch as S
+end box
+box "DFSP A"
+    participant "DFSP A\n(Payer)" as D2
+end box
+box "DFSP B"
+    participant "DFSP B\n(Payee)" as D3
+end box
+
+... PISP has initiated Thirdparty Transaction Request with ""POST /thirdpartyRequests/transactions""...
+
+D2 -> D2: Generate a unique transactionId for this transaction request:\n**""11111111-0000-0000-0000-000000000000""**
+
+
+rnote left of D2 #Light
+**""PUT /thirdpartyRequests/transactions""**
+**""    /00000000-0000-0000-0000-000000000000""**
+""FSPIOP-Source: dfspa""
+""FSPIOP-Destination: pispa""
+{
+  "transactionId": "11111111-0000-0000-0000-000000000000",
+  "transactionRequestState": "RECEIVED"
+}
+end note
+D2 -> S: ""PUT /thirdpartyRequests/transactions""\n""    /00000000-0000-0000-0000-000000000000""
+S --> D2: ""200 OK""
+S -> D1: ""PUT /thirdpartyRequests/transactions""\n""    /00000000-0000-0000-0000-000000000000""
+D1 --> S: ""200 OK""
+
+rnote left of D2 #Light
+**""POST /quotes""**
+""FSPIOP-Source: dfspa""
+""FSPIOP-Destination: dfspb""
+{
+  "quoteId": "22222222-0000-0000-0000-000000000000",
+  "transactionId": "11111111-0000-0000-0000-000000000000",
+  "transactionRequestId": "00000000-0000-0000-0000-000000000000",
+  "payee": {
+    "partyIdInfo": {
+      "partyIdType": "MSISDN",
+      "partyIdentifier": "+4412345678",
+      "fspId": "dfspb"
+    }
+  },
+  "payer": {
+    "personalInfo": {
+      "complexName": {
+        "firstName": "Ayesha",
+        "lastName": "Takia"
+      }
+    },
+    "partyIdInfo": {
+      "partyIdType": "THIRD_PARTY_LINK",
+      "partyIdentifier": "qwerty-56789",
+      "fspId": "dfspa"
+    },
+    },
+  },
+  "amountType": "SEND",
+  "amount": {
+    "amount": "100",
+    "currency": "USD"
+  },
+  "transactionType": {
+    "scenario": "TRANSFER",
+    "initiator": "PAYER",
+    "initiatorType": "CONSUMER"
+  },
+  "note": "quote note"
+}
+end note
+D2 -> S: ""POST /quotes""
+S --> D2: ""202 Accepted""
+S -> D3: ""POST /quotes""
+D3 --> S: ""202 Accepted""
+
+D3 -> D3: Quote fails for some reason.
+
+rnote left of D3 #Light
+**""PUT /quotes/22222222-0000-0000-0000-000000000000/error""**
+""FSPIOP-Source: dfspb""
+""FSPIOP-Destination: dfspa""
+{
+ "errorInformation": {
+    "errorCode": "XXXX",
+    "errorDescription": "XXXX",
+    "extensionList": []
+  }
+}
+end note
+D3 -> S: ""PUT /quotes/22222222-0000-0000-0000-000000000000/error""
+S --> D3: ""200 OK""
+S -> D2: ""PUT /quotes/22222222-0000-0000-0000-000000000000/error""
+D2 --> S: ""200 OK""
+
+note left of D2
+  Quote failed, DFSP needs to inform PISP
+end note
+
+rnote left of D2 #Light
+**""PUT /thirdpartyRequests/transactions""**
+**""    /00000000-0000-0000-0000-000000000000/error""**
+""FSPIOP-Source: dfspa""
+""FSPIOP-Destination: pispa""
+{
+  "errorInformation": {
+    "errorCode": "6XXX",
```

Comment:
to report PISP errors we reserved 7xxx  code range.  DFSP probably will not want to share detailed quote error with PISP so maybe we should point out here that it's DFSP responsibility to send proper 

already defined error codes: https://github.com/mojaloop/sdk-standard-components/blob/master/src/lib/errors/index.js#L99-L110

PISP errorCode.
```suggestion
    "errorCode": "71xx",
```